src/status_im/ui/screens/chat/views.cljs

```diff
@@ -484,23 +479,21 @@
       (let [{:keys [chat-id show-input? group-chat admins invitation-admin] :as chat}
             ;;we want to react only on these fields, do not use full chat map here
             @(re-frame/subscribe [:chats/current-chat-chat-view])
-            mutual-contact-requests-enabled? @(re-frame/subscribe [:mutual-contact-requests/enabled?])
-            max-bottom-space (max @bottom-space @panel-space)]
+            max-bottom-space                                                          (max @bottom-space
```

Comment:
This is a good example where the code formatter creates something unreadable IMO. If you forcefully add line breaks between every destructured key, it'll format to something easier to the eyes:

```clojure
(let [{:keys [chat-id
              show-input?
              group-chat
              admins
              invitation-admin]
       :as   chat}
      @(re-frame/subscribe [:chats/current-chat-chat-view])
      max-bottom-space (max @bottom-space
                            @panel-space)]
  ...)
```